<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天天静听</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/ico.ico</normaloff>:/images/ico.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>播放方式</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>顺序播放</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>单曲循环</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>随机播放</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Agency FB</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>更多设置</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonSet">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="labelPlayStatus">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>正在播放：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelMusicName">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>歌曲名</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QSlider" name="horizontalSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelTime">
            <property name="font">
             <font>
              <family>SimSun-ExtB</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>00:00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QToolButton" name="toolButtonPre">
            <property name="toolTip">
             <string>上一曲</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/images/gtk-media-next-rtl.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonPlayOrPause">
            <property name="toolTip">
             <string>播放或暂停</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/images/gtk-media-play-ltr.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonNext">
            <property name="toolTip">
             <string>下一曲</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/images/gtk-media-next-ltr.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonStop">
            <property name="toolTip">
             <string>停止</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/images/gtk-media-stop.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonAddMusic">
            <property name="toolTip">
             <string>添加歌曲列表</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/images/gtk-open.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonExit">
            <property name="toolTip">
             <string>退出</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/images/exit.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelVolume">
          <property name="text">
           <string>音量:50</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
